crux_cli/CHANGELOG.md

```diff
@@ -7,6 +7,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.2.0](https://github.com/redbadger/crux/releases/tag/crux_cli-v0.2.0) - 2025-09-02
+
+Breaking changes:
+
+The `--swift` and `--kotlin` arguments to `bindgen` now accept an optional output directory:
```

Comment:
If I'm reading the code correctly, the presence or absence of the argument also decides whether or not the bindings for the language get generated? 

We should probably be more explicit about that - it's not that they accept optional output directory, it's that generating the code is optional, and requires an ouput path when selected.